diff --git a/packages/scripts/src/commands/git-commit.ts b/packages/scripts/src/commands/git-commit.ts
index 35c0df56..16202e85 100644
--- a/packages/scripts/src/commands/git-commit.ts
+++ b/packages/scripts/src/commands/git-commit.ts
@@ -78,7 +78,6 @@ export async function gitCommitVerify(lang: Lang = 'en-us', ignores: RegExp[] =
 
   if (!REG_EXP.test(commitMsg)) {
     const errorMsg = locales[lang].gitCommitVerify;
-
-    throw new Error(errorMsg);
+    console.warn(errorMsg);
   }
 }
